spec/features/avo/divider_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.feature "Divider", type: :feature do
+  let!(:user) { create :user }
+
+  before do
+    visit "/admin/resources/users"
+    click_on "Actions"
+    @parsed_body = Nokogiri::HTML(page.body)
+    @dividers = @parsed_body.css(".relative.col-span-full.border-t")
```

Comment:
We don't need to parse the response like so with Nokogiri. capybara/rspec should have something already.

```ruby
expect(page).to have SOMETHING

within(find("SOME_SELECTOR")) do
  find("SOMETHING_ELSE")
end
```

Please check other feature tests.

Also, let's ensure the dividers and actions are in order. So we're not checking just for presence on the page. 

[This](https://www.betterspecs.org/) is a good resource aboout how to write better specs. It doesn't necesarily have guides on selectors, but you can see some best practices.